src/LogTable.js

```diff
@@ -35,42 +35,187 @@ const Styles = styled.div`
       }
     }
   }
+
+  .logtable-head {
+    display: flex;
+  }
+  .logtable-row:hover {
+    background-color: #e6e6e6;
```

Comment:
just making sure - these are from designs?

Reply:
No, I added this for a better user experience.